dojo/tools/qualys_webapp/parser.py

```diff
@@ -73,24 +81,32 @@ def attach_unique_extras(endpoints, requests, responses, finding, date, qid, par
         port = ""  # Set port to empty string by default
         # Split the returned network address into host and
         try:  # If there is port number attached to host address
-            host, port = parsedUrl.netloc.split(':')
-        except:  # there's no port attached to address
+            host, port = parsedUrl.netloc.split(":")
+        except BaseException:  # there's no port attached to address
```

Comment:
Change to `Exception`

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7